readme.txt

```diff
@@ -242,6 +242,33 @@ Yes, through the use of our commercial addon called [Recurring Payments](https:/
 13. Pay with PayPal modal
 
 == Changelog ==
+
+= 2.11.4, December 27, 2021 =
+* New: Add EDD notifications to EDD screens.
+* New: Add product education to EDD settings screens.
+* New: Add dismissible review request to dashboard, EDD settings screens.
```

Comment:
This should be focused on the reason, which is to give us the ability to have users give us feedback, since this dismissible item can result in a feature request submission.